src/borg/archiver.py

```diff
@@ -507,39 +507,60 @@ def create_inner(archive, cache, fso):
                 except OSError:
                     pass
             logger.debug('Processing files ...')
-            for path in args.paths:
-                if path == '-':  # stdin
-                    path = args.stdin_name
-                    if not dry_run:
-                        try:
-                            status = fso.process_stdin(path=path, cache=cache)
-                        except BackupOSError as e:
-                            status = 'E'
-                            self.print_warning('%s: %s', path, e)
-                    else:
-                        status = '-'
-                    self.print_file_status(status, path)
-                    continue
-                path = os.path.normpath(path)
-                parent_dir = os.path.dirname(path) or '.'
-                name = os.path.basename(path)
-                # note: for path == '/':  name == '' and parent_dir == '/'.
-                # the empty name will trigger a fall-back to path-based processing in os_stat and os_open.
-                with OsOpen(path=parent_dir, flags=flags_root, noatime=True, op='open_root') as parent_fd:
+            if args.content_from_command:
```

Comment:
the overall diff is not very readable, it looks like more change than it really is (i guess).

could you change the if-condition to `if not args.content_from_command` and swap the if/else blocks to see if the github diff display works better then?

Reply:
Yes, it looks like far more than it is. However, swapping the branches isn't much of an improvement [(swapped version)](https://github.com/borgbackup/borg/compare/9019b48fd5982802af44a510b9decc05fa672c33...pgerber:swapped#diff-48f2b9140a35ffd45a46ccfbe63931a6L509).

Kind of sad that after all this years GitHub's diff view has improved so little.

Reply:
ok, I try to review "as is".